app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -627,6 +628,8 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
             tabViewSetting.set(settings.getTabViewPingString())
             closeTabSetting.set(settings.getTabTimeoutPingString())
 
+            inactiveTabsCount.set(browserStore.state.inactiveTabs.size.toLong())
```

Comment:
@Mugurell @MozillaNoah Is the intention for the probe to see what tabs _could_ be considered inactive? This extension function does not take into account the settings to see if inactive tabs is enabled first, so this value can be reported with a non-zero number incorrectly.

If not, a quick fix would be to use [this extension function](https://github.com/mozilla-mobile/fenix/blob/8b8fb4f00d48363e42faf497fad20dd852eeb8f5/app/src/main/java/org/mozilla/fenix/tabstray/browser/TabSorter.kt#L78-L85) instead if you remove the `private` visibility modifier.

Reply:
Thank you @jonalmeida! Makes sense.
Thinking about making another extension to BrowserState then for `actualInactivetabs` since this method will be called from the Application and rename the current `inactiveTabs` to `potentialInactiveTabs`.